### PR TITLE
mappings for member names in XMLAttribute

### PR DESCRIPTION
**Pull request by djencks**

### CODE REVIEW

File: biz.aQute.bndlib/src/aQute/bnd/annotation/xml/XMLAttribute.java [hunk 1]
```diff
@@ -32,12 +35,38 @@
...
 	};
 
+	/**
+	 * Array of Strings of the form key=value where the key is the name of a
+	 * member and the value is the xml attribute name to use for that member's
+	 * value. For instance, "value=simple" causes @Simple("foo") to emit
+	 * ns:simple="foo".
+	 * 
+	 * @return Array of member-name to xml-attribute-name mappings.
+	 */
```
Comment (bjhargrave) [inline biz.aQute.bndlib/src/aQute/bnd/annotation/xml/XMLAttribute.java]: @since 1.1

**Comment by bjhargrave** [comment]
Also need Signed-off-by in commit message.

**Comment by bjhargrave** [comment]
LGTM

**Comment by djencks** [comment]
This adds 
{code}
	/**
	 * Array of Strings of the form key=value where the key is the name of a
	 * member and the value is the xml attribute name to use for that member's
	 * value. For instance, "value=simple" causes @Simple("foo") to emit
	 * ns:simple="foo".
	 * 
	 * @return Array of member-name to xml-attribute-name mappings.
	 */
	String[] mapping() default {};
{code}
to XMLAttribute, along with the supporting code.

**Pull request merged**